### PR TITLE
Adding new translation for Pokémon Red and Blue Setup

### PR DESCRIPTION
**Pull request by TheLynk**
Please format your title with what portion of the project this pull request is
targeting and what it's changing.

ex. "MyGame4: implement new game" or "Docs: add new guide for customizing MyGame3"

What is this fixing or adding?
Adding new translation for Pokémon Red and Blue Setup

How was this tested?
in local web server

If this makes graphical changes, please attach screenshots.
no graphical changes

### CODE REVIEW

File: worlds/pokemon_rb/__init__.py [hunk 1]
```diff
@@ -67,6 +67,15 @@ class PokemonWebWorld(WebWorld):
...
+    setup_es = Tutorial(
+        setup_en.tutorial_name,
+        setup_en.description,
+        "Français",
+        "setup_fr.md",
+        "setup/fr",
+        ["TheLynk"]
+    )
+
     tutorials = [setup_en, setup_es]
```
Comment (Bicoloursnake) [inline worlds/pokemon_rb/__init__.py:80]: I'm not sure how to recommend changes so that you can just click a button and implement them, but I ran a local test, and this overwrote the Spanish Tutorial. The 'setup_es' Tutorial should instead probably be 'setup_fr', and setup_fr should be added to tutorials on line 79.

**Review by Bicoloursnake — CHANGES_REQUESTED** [review]

File: worlds/pokemon_rb/__init__.py [hunk 2]
```diff
@@ -67,7 +67,16 @@ class PokemonWebWorld(WebWorld):
         ["Shiny"]
     )
 
-    tutorials = [setup_en, setup_es]
+    setup_es = Tutorial(
```
Comment (Bicoloursnake) [inline worlds/pokemon_rb/__init__.py:70]: This line should be `setup_fr = Tutorial(`, but otherwise looks good (at least for this part, I can't read French to review the documentation itself).

File: worlds/pokemon_rb/docs/setup_fr.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+## Logiciel requis
+
+- BizHawk : [Version BizHawk de TASVideos](https://tasvideos.org/BizHawk/ReleaseHistory)
+   - Les versions 2.3.1 et ultérieures sont prises en charge. La version 2.7 est recommandée pour des raisons de stabilité.
+   - Des instructions d'installation détaillées pour BizHawk peuvent être trouvées sur le lien ci-dessus.
+   - Les utilisateurs Windows doivent d'abord exécuter le programme d'installation des prérequis, qui peut également être trouvé sur le lien ci-dessus.
+- Le client Archipelago intégré, qui peut être installé [ici](https://github.com/ArchipelagoMW/Archipelago/releases)
+   (sélectionnez `Pokemon Client` lors de l'installation).
+- Fichiers ROM Pokémon Rouge et/ou Bleu. La communauté de l'Archipelago ne peut pas les fournir.
```
Comment (ReverM) [inline worlds/pokemon_rb/docs/setup_fr.md:15]:
```suggestion
- Fichiers ROM Pokémon Rouge et/ou Bleu. La communauté Archipelago ne peut pas les fournir.
```
Small nitpick, might be wrong

File: worlds/pokemon_rb/docs/setup_fr.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+-[Pokémon Red and Blue Archipelago Map Tracker](https://github.com/j-imbo/pkmnrb_jim/releases/latest), à utiliser avec [PopTracker](https://github.com/black-sliver/PopTracker/releases)
+
+
+## Configuration de BizHawk
+
+Une fois BizHawk installé, ouvrez EmuHawk et modifiez les paramètres suivants :
+
+- (≤ 2,8) Allez dans Config > Personnaliser. Passez à l'onglet Avancé, puis faites passer le Lua Core de "NLua+KopiLua" à "Lua+LuaInterface". 
+Puis redémarrez EmuHawk. Ceci est nécessaire pour que le script Lua fonctionne correctement.
```
Comment (ReverM) [inline worlds/pokemon_rb/docs/setup_fr.md:27]:
```suggestion
, puis redémarrez EmuHawk. Ceci est nécessaire pour que le script Lua fonctionne correctement.
```
Sentence starting with "Puis" feels weird

**Comment by ReverM** [comment]
Could the name of the PR also specify that it's a french translation?

**Comment by ReverM** [comment]
There is a failed check, the rest of the doc is perfect.

File: worlds/pokemon_rb/docs/setup_fr.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
+# Guide de configuration pour Pokémon Rouge et Bleu : Archipelago
+
+## Important
+
+Comme nous utilisons BizHawk, ce guide s'applique uniquement aux systèmes Windows et Linux.
+
+## Logiciel requis
+
+- BizHawk : [Version BizHawk de TASVideos](https://tasvideos.org/BizHawk/ReleaseHistory)
+   - Les versions 2.3.1 et ultérieures sont prises en charge. La version 2.7 est recommandée pour des raisons de stabilité.
```
Comment (Alchav) [inline worlds/pokemon_rb/docs/setup_fr.md:10]: I just opened a pull request to change the recommended Bizhawk version to 2.9 in the existing setup guides

Comment (Alchav) [inline worlds/pokemon_rb/docs/setup_fr.md:10]:
```suggestion
   - Les versions 2.3.1 et ultérieures sont prises en charge. La version 2.9 est recommandée.
```
don't know the language but used Google Translate for this

File: worlds/pokemon_rb/docs/setup_fr.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
+# Guide de configuration pour Pokémon Rouge et Bleu : Archipelago
+
+## Important
+
+Comme nous utilisons BizHawk, ce guide s'applique uniquement aux systèmes Windows et Linux.
+
+## Logiciel requis
+
+- BizHawk : [Version BizHawk de TASVideos](https://tasvideos.org/BizHawk/ReleaseHistory)
+   - Les versions 2.3.1 et ultérieures sont prises en charge. La version 2.9 est recommandée.
```
Comment (Alchav) [inline worlds/pokemon_rb/docs/setup_fr.md:10]:
```suggestion
   - Les versions 2.3.1 et ultérieures sont prises en charge. La version 2.9.1 est recommandée.
```
This was updated again just now, whoopsie

File: worlds/pokemon_rb/docs/setup_fr.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+## Important
+
+Comme nous utilisons BizHawk, ce guide s'applique uniquement aux systèmes Windows et Linux.
+
+## Logiciel requis
+
+- BizHawk : [Version BizHawk de TASVideos](https://tasvideos.org/BizHawk/ReleaseHistory)
+   - Les versions 2.3.1 et ultérieures sont prises en charge. La version 2.9.1 est recommandée.
+   - Des instructions d'installation détaillées pour BizHawk peuvent être trouvées sur le lien ci-dessus.
```
Comment (ReverM) [inline worlds/pokemon_rb/docs/setup_fr.md:11]:
```suggestion
   - Des instructions d'installation détaillées pour BizHawk peuvent être trouvées via le lien ci-dessus.
``` 
I'm not sure if this is better, but "on" feels off

File: worlds/pokemon_rb/docs/setup_fr.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+   - Les utilisateurs Windows doivent d'abord exécuter le programme d'installation des prérequis, qui peut également être trouvé sur le lien ci-dessus.
+- Le client Archipelago intégré, qui peut être installé [ici](https://github.com/ArchipelagoMW/Archipelago/releases)
+   (sélectionnez `Pokemon Client` lors de l'installation).
```
Comment (ReverM) [inline worlds/pokemon_rb/docs/setup_fr.md:14]:
```suggestion 
```
When installing archipelago, you do not have the option to select specific client anymore.

File: worlds/pokemon_rb/docs/setup_fr.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+- Fichiers ROM Pokémon Rouge et/ou Bleu. La communauté Archipelago ne peut pas les fournir.
+
+## Logiciel en option
```
Comment (ReverM) [inline worlds/pokemon_rb/docs/setup_fr.md:17]: I've used "Logiciels facultatifs" for my witness guide, that felt more good. That's purely a stylistic comment

File: worlds/pokemon_rb/docs/setup_fr.md [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+
+-[Pokémon Red and Blue Archipelago Map Tracker](https://github.com/j-imbo/pkmnrb_jim/releases/latest), à utiliser avec [PopTracker](https://github.com/black-sliver/PopTracker/releases)
+
+
+## Configuration de BizHawk
+
+Une fois BizHawk installé, ouvrez EmuHawk et modifiez les paramètres suivants :
+
+- (≤ 2,8) Allez dans Config > Personnaliser. Passez à l'onglet Avancé, puis faites passer le Lua Core de "NLua+KopiLua" à "Lua+LuaInterface". 
+Ensuite redémarrez EmuHawk. Ceci est nécessaire pour que le script Lua fonctionne correctement.
```
Comment (ReverM) [inline worlds/pokemon_rb/docs/setup_fr.md:27]:
```suggestion
Ensuite, redémarrez EmuHawk. Cela est nécessaire pour que le script Lua fonctionne correctement.
```

File: worlds/pokemon_rb/docs/setup_fr.md [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+
+### Qu'est-ce qu'un fichier YAML et pourquoi en ai-je besoin ?
+
+Votre fichier YAML contient un ensemble d'options de configuration qui fournissent au générateur des informations sur la manière dont il doit
+générer votre jeu. Chaque joueur d'un multimonde fournira son propre fichier YAML. Cette configuration permet à chaque joueur de profiter
+une expérience personnalisée à leur goût, et les différents joueurs du même multimonde peuvent tous avoir des options différentes.
+
+### Où puis-je obtenir un fichier YAML ?
+
+Vous pouvez générer un yaml ou télécharger un modèle en visitant la [Page des paramètres du joueur Pokémon Rouge et Bleu](/games/Pokemon%20Red%20and%20Blue/player-settings)
```
Comment (ReverM) [inline worlds/pokemon_rb/docs/setup_fr.md:49]: Player settings to be changed to player options on next release
"Page des options de joueur Pokémon Red and Blue" could be a good translation

File: worlds/pokemon_rb/docs/setup_fr.md [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+
+Il est important de noter que l'option `game_version` détermine le fichier ROM qui sera corrigé.
```
Comment (ReverM) [inline worlds/pokemon_rb/docs/setup_fr.md:51]: Later, "patch" is mentionned in "pokémon patch file". I'm not sure of the word "corrigé" here.

File: worlds/pokemon_rb/docs/setup_fr.md [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+Le joueur et la personne qui génère (s'ils génèrent localement) auront besoin du fichier ROM correspondant.
```
Comment (ReverM) [inline worlds/pokemon_rb/docs/setup_fr.md:52]:
```suggestion
Le joueur et la personne qui génère (si elle génère localement) auront besoin du fichier ROM correspondant.
```

File: worlds/pokemon_rb/docs/setup_fr.md [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+-[Pokémon Red and Blue Archipelago Map Tracker](https://github.com/j-imbo/pkmnrb_jim/releases/latest), à utiliser avec [PopTracker](https://github.com/black-sliver/PopTracker/releases)
+
+
+## Configuration de BizHawk
+
+Une fois BizHawk installé, ouvrez EmuHawk et modifiez les paramètres suivants :
+
+- (≤ 2,8) Allez dans Config > Personnaliser. Passez à l'onglet Avancé, puis faites passer le Lua Core de "NLua+KopiLua" à "Lua+LuaInterface". 
+Ensuite redémarrez EmuHawk. Ceci est nécessaire pour que le script Lua fonctionne correctement.
+   **REMARQUE : Même si « Lua+LuaInterface » est déjà sélectionné, basculez entre les deux options et resélectionnez-la. Nouvelles installations**
```
Comment (ReverM) [inline worlds/pokemon_rb/docs/setup_fr.md:28]: This paragraph is really hard to parse. It might be worth to just rewrite it in a different way to have the same message. I think the structure works in english but not at all in french (for instance, starting with "Nouvelles installations" is gramatically odd)

File: worlds/pokemon_rb/docs/setup_fr.md [hunk 15]
```diff
@@ -0,0 +1,102 @@
...
+   **des versions plus récentes d'EmuHawk ont tendance à afficher "Lua+LuaInterface" comme option sélectionnée par défaut mais se chargent toujours**
+   ** "NLua+KopiLua" jusqu'à ce que cette étape soit terminée.**
+- Sous Config > Personnaliser > Avancé, assurez-vous que la case AutoSaveRAM est cochée et cliquez sur le bouton 5s.
+   Cela réduit la possibilité de perdre des données de sauvegarde en cas de panne de l'émulateur.
+- Sous Config > Personnaliser, cochez la case "Exécuter en arrière-plan". Cela empêchera la déconnexion du client pendant EmuHawk s'exécute en arrière-plan.
+
+Il est fortement recommandé d'associer des extensions de rom GB (\*.gb) à l'EmuHawk que nous venons d'installer.
```
Comment (ReverM) [inline worlds/pokemon_rb/docs/setup_fr.md:35]: I see that is how it is written in the original guide, but the switch from "you" to "we" feels odd.

File: worlds/pokemon_rb/docs/setup_fr.md [hunk 16]
```diff
@@ -0,0 +1,102 @@
...
+* `<MN>`
+* `<MALE>` alias pour `♂`
+* `<FEMALE>` alias pour `♀`
+
+## Rejoindre un jeu multimonde
+
+### Obtenez votre fichier de patch Pokémon
+
+Lorsque vous rejoignez un jeu multimonde, il vous sera demandé de fournir votre fichier YAML à celui qui l'héberge. Une fois cela fait,
+l'hébergeur vous fournira soit un lien pour télécharger votre fichier de données, soit un fichier zip contenant les données de chacun
```
Comment (ReverM) [inline worlds/pokemon_rb/docs/setup_fr.md:75]:
```suggestion
l'hôte vous fournira soit un lien pour télécharger votre fichier de données, soit un fichier zip contenant les données de chacun
```

File: worlds/pokemon_rb/docs/setup_fr.md [hunk 17]
```diff
@@ -0,0 +1,102 @@
...
+des dossiers. Votre fichier de données doit avoir une extension « .apred » ou « .apblue ».
```
Comment (ReverM) [inline worlds/pokemon_rb/docs/setup_fr.md:76]:
```suggestion
des dossiers. Votre fichier de données devrait avoir une extension `.apred` ou `.apblue`.
```

**Comment by ReverM** [comment]
The english doc is not quite up to date. It might be worth waiting on it to update before carrying over the translation.
The overall structure is somewhat unnatural at times, it might be worth trying to check if some of the sentences could be rewritten in a way that doesn't preserve english structure but preserve the message conveyed by the sentences (the one about fresh installs is one i can think of).

**Comment by BootsinSoots** [comment]
@TheLynk I see that the english guide was updated since Rever's last comment. If you're still interested in working on this, the english version appears to be in a stable state.
